### PR TITLE
equationorousious tableoumus

### PR DESCRIPTION
**Pull request by t-lohse**
lol

### CODE REVIEW

**Comment by t-lohse** [comment]
@Laursen79 @JE-FH this PR is long overdue for a review. Please get to it

**Review by Laursen79 — CHANGES_REQUESTED** [review]
Please Remove .idea

**Comment by t-lohse** [comment]
https://plugins.jetbrains.com/plugin/18321-spotless-gradle lol

File: src/main/java/org/cs23sw612/ExampleSUL.java
Comment (Laursen79) [inline src/main/java/org/cs23sw612/ExampleSUL.java:1]: :;(

File: src/main/java/org/cs23sw612/Interfaces/OutputListener.java
Comment (Laursen79) [inline src/main/java/org/cs23sw612/Interfaces/OutputListener.java:1]: This should just be deleted

Comment (t-lohse) [reply src/main/java/org/cs23sw612/Interfaces/OutputListener.java:1]: dude, det er legit din fil

**Comment by t-lohse** [comment]
@Laursen79 pls

**Comment by t-lohse** [comment]
REEE

**Review by Laursen79 — APPROVED** [review]

**Pull request merged**